jupyterhub/user.py

```diff
@@ -96,7 +104,8 @@ def _db_changed(self, change):
         db = change.new
         if self._user_id is not None:
             self.orm_user = db.query(orm.User).filter(orm.User.id == self._user_id).first()
-        self.spawner.db = change.new
+        for name, spawner in self.spawners.items():
```

Comment:
Line 106: The query after new session gets the user from the db in 106 and sets the instance's orm_user to the result of the query. 

Line 107: Iterates and gets the name and spawner for each entry in the spawner dictionary for a user.

Line 108: While iterating, get the named spawner that corresponds to the user and name in the db dictionary of user's spawners. So `spawner.orm_spawner` is holding ???